addon/templates/components/from-elsewhere.hbs

```diff
@@ -1,9 +1,11 @@
 {{#if initialized}}
-  {{#if hasBlock}}
-    {{yield (get (get service.actives name) 'lastObject.component') }}
-  {{else}}
-    {{#with (get (get service.actives name) 'lastObject.component') as |c|}}
-      {{component c}}
-    {{/with}}
-  {{/if}}
+  {{#let (get service.actives name) as |active| }}
+    {{#if hasBlock}}
+      {{yield active.lastObject.component active.lastObject.outsideParams}}
+    {{else}}
+      {{#with active.lastObject.component as |c|}}
+        {{component c outsideParams=active.lastObject.outsideParams}}
```

Comment:
I don't think we should pass the outsideParams to the component. It confuses what they're for.

Also, if people *really* need them in the component they're still able to do that using the block form of `from-elsewhere`.

Reply:
removed